templates/projects/_helpers.html

```diff
@@ -293,6 +293,24 @@ <h2><strong> {{_('Overall progress')}}</strong>: {{ overall_progress | round | i
 </div>
 {% endmacro %}
 
+{% macro render_project_incomplete_info_panel(info) %}
+<div class="panel panel-info">
+    <div class="panel-heading">
+        <h3 class="panel-title"><i class="fa fa-info-circle"></i> {{ _('You have no remaining tasks, but the project is not complete because:') }}</h3>
+    </div>
+    <div class="panel-body">
+        <ul>
+            {% if 'user_preferences' in info %}
+            {% set task_scheduler = info['user_preferences'].get('task_scheduler') %}
```

Comment:
If the goal is to identify the type of scheduler the project has, it can be obtained from
` info["sched"]` whose value to be checked in. this scenario could be either `user_pref_scheduler` or `task_queue_scheduler`

Reply:
We want the human-friendly name. I don’t think this is included in the info dictionary. For context, see https://github.com/bloomberg/pybossa/pull/866